.changeset/giant-years-drum.md

```diff
@@ -0,0 +1,6 @@
+---
+"@sveltejs/kit": patch
+---
+
+fix: CSP conflicts with sha/nonce during dev
+feat: add CSP support for style-src-elem 
```

Comment:
if we have two things being done in a PR we create two changesets rather than putting two messages in a single changeset